.gitmodules

```diff
@@ -64,3 +64,9 @@
 [submodule "vendor/golang.org/x/crypto"]
 	path = vendor/golang.org/x/crypto
 	url = https://go.googlesource.com/crypto
+[submodule "vendor/github.com/syndtr/goleveldb"]
```

Comment:
what research did we do before choosing LevelDB as our production solution? the fact that we used it in the alpha is not enough. 

1. did we compare it to other solutions? we have a very distinct use case.. 99% of our block data will almost never be accessed after it's stored

2. what are other blockchain projects using for their block storage?

3. I'll be happy to see benchmarks that show how well LevelDB handles TB of data. If I compare our block storage to analytics or logging solutions, they usually use file system to save the old logs.. why not go to this type of storage solutions? If we can't find any suitable benchmarks that we can run, maybe take the time and implement one ourselves